apps/docs/src/app/fn/component-docs/utilities/fn-disabled/fn-disabled-docs.component.html

```diff
@@ -0,0 +1,52 @@
+<fd-docs-section-title id="simple" componentName="FnDisabled">Basic example</fd-docs-section-title>
+<description>
+    <p>
+        <code>[fnDisabled]</code> is implementation of default view modifier. When passed true or used as an attribute
+        to <b>any</b> element, which is rendered in DOM will have similar traits. <code>[fnDisabled]</code> will add
+        <code>.is-disabled</code>, <code>[disabled]</code> and <code>[aria-disabled]</code> to the target element.
+    </p>
+    <p>
+        <code>[fnDisabled]</code> is independent directive and parent or child directives do not have any impact on it.
+        Everything is quite opposite.
+    </p>
+</description>
+<component-example [hasBackground]="false">
+    <fundamental-ngx-fn-disabled-example></fundamental-ngx-fn-disabled-example>
+</component-example>
+<code-example [exampleFiles]="defaultExample"></code-example>
+
+<fd-docs-section-title id="di" componentName="FnDisabled">FnDisabledProvider</fd-docs-section-title>
+<description>
+    <p>
+        There are cases when you need to tap into the process of declaring certain element disabled. For example, you
+        want to add class <code>is_very_disabled</code>&nbsp;<u>alongside</u> default behavior when disabled and remove
+        when not. You can achieve that two ways, one recommended and one side effect of having notifier. We will gonna
+        go through recommended way in below.
+    </p>
+    <p>Things to note:</p>
+    <ul>
+        <li>
+            <code>FnDisabledProvider</code> will also detect native HTML <code>[disabled]</code>
+            attribute and will act upon it's change.
+        </li>
+        <li>
+            <code>FnDisabledProvider</code> will detect if any ancestor has <code>[fnDisabled]="true"</code> and will
+            also set itself as <b>disabled</b>
```

Comment:
This line contradicts a statement made earlier above, that `fnDisabled` is independent of parent and child directives

Reply:
No. FnDisabled by itself will not set disabled state when parent changes, but when component has FnDisabledProvider in it, then it will look through dep tree and FnDisabled will have effect on other elements. If component has BOTH FnDisabledProvider and [fnDisabled], then FnDisabledProvider will take effect

Reply:
Ok now I understand, thanks

Reply:
It is nicely visible in FnDisabledProvider implementation. It depends on parent and self injection tokens, but [fnDisabled] does not have any deps, it just adds disabled state default. That's one of the use cases of the FnDisabledProvider, when you want to both watch for outside changes and respect parent changes and act upon their arrival. I plan to add configuration capability to that Provider, so that you'll be able to customize behavior of that service, even add ability to exclude directive from update sequence. But that's for the future

Reply:
In general provider can be used without directive and directive can be used with or without provider. Those are just different sides of the feature. One more concerned on self and nothing else and second one is concerned about everything, even native updates